core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/extensions.scala

```diff
@@ -185,3 +191,11 @@ case class Lateral(expr: LogicalPlan) extends UnaryNode {
   override def child: LogicalPlan = expr
   override def output: Seq[Attribute] = expr.output
 }
+
+// TSQL has some INSERT and MERGE actions that are extensions over that which DataBricks SQL supports. These
+// can perhaps be translated by the generator.
+// TODO: Should these be in a TSQl only file?
+
+case class InsertDefaultsAction(condition: Option[Expression]) extends MergeAction {
```

Comment:
missing a rule (or generator)

Reply:
I need to check Databricks SQL a bit more to see if there is a way to do this. At first glance there is not. When used in TSQL, there is not column list. So I think it cannot be translated and I need a rule

Reply:
Rule to trap this and indicate it is not supported added. There is equivalent in Databricks SQL and it cannot be simulated as the equivalent  of the TSQL:

```tsql
MERGE x INTO y ON x.a = y.a WHEN NOT MATCHED INSERT DEFAULT VALUES
```

Would require us to know the column names, and they are not listed in the TSQL. It woudl be:

```sql
MERGE x INTO y ON x.a = y.a WHEN NOT MATCHED INSERT (c1, c2, c3) VALUES (DEFAULT, DEFAULT, DEFAULT)
```